cg/cli/reset.py

```diff
@@ -2,6 +2,11 @@
 from typing import Iterable, Optional
 
```

Comment:
What do you think about moving this to the delete sub command instead? We are deleting a case from loqusDB (main purpose) and as a side effect we also need to remove the loqusDB ID.

Reply:
Yes, sounds good!

Tagging @fevac